documentation/long-stay-baseline.mdx

```diff
@@ -0,0 +1,528 @@
+
+A series of Jupyter Notebooks used to generate this report are available on [Github](https://github.com/nhsx/skunkworks-long-stayer-risk-stratification-baseline/tree/main/notebooks).
+
+## Table of contents
+
+* 1. [Background](#background)
+* 2. [Approach](#approach)
+* 3. [Data ingest and processing](#dataingestandprocessing)
+* 4. [Feature engineering](#featureengineering)
+* 5. [Statistical analysis](#statisticalanalysis)
+* 6. [Modelling](#modelling)
+    * 6.1 [Regression models](#regressionmodels)
+    * 6.2 [Demographic analysis](#demographicanalysis)
+    * 6.3 [Classification models](#classificationmodels)
+    * 6.4 [Model comparison](#modelcomparison)
+* 7. [Conclusions](#conclusions)
+* 8. [Future work](#futurework)
+
+## 1. Background<a name='background'></a>
+
+Hospital long stayers, those with a <a href='https://www.england.nhs.uk/urgent-emergency-care/reducing-length-of-stay/' target='_blank'>length of stay (LoS) of 21 days or longer</a>, have significantly worse medical and social outcomes than other patients. Long-stayers are often medically optimised (fit for discharge) many days before their actual discharge. Moreover, there are a complex mixture of medical, cultural and socioeconomic factors which contribute to the causes of unnecessary long stays.
+
+This project aims to complement [previous work](long-stay) by generating simple baseline regression and classification models that could be replicated at other hospital trusts, and is divided into two phases:
+
+1. Series of Jupyter Notebooks containing baseline model code
+2. [Reproducible Analytical Pipeline](https://github.com/NHSDigital/rap-community-of-practice) including data pipelines
+
+Currently, this project has completed **Phase 1**.
+
+## 2. Approach<a name='approach'></a>
+
+The aim of this project is to perform the simplest possible feature engineering and modelling to arrive at a reasonable baseline model, for more advanced feature engineering and modelling to be compared against.
+
+The approach involved:
+
+1. Defining the population and data cleaning
+2. Feature engineering, focussing on basic numerical and categorical features
+3. Simple baseline models implemented using commonly available packages including [scikit-learn 1.1.1](https://scikit-learn.org/), [CatBoost 1.0.6](https://catboost.ai) and [XGBoost 1.3.3](https://xgboost.readthedocs.io/en/stable/)
+4. Analysis of model performance by demographic
+5. A comparison of regression-based and classification-based risk stratification models
+6. A set of extensions for future work
+
+## 3. Data ingest and processing<a name='dataingestandprocessing'></a>
+
+GHNHSFT performed a SQL export from their EPR system containing ~770,000 records across 99 columns, with significant sparsity across several columns and a section of rows, as visualised by light coloured blocks (null values) in the below image:
+
+![Image of data sparsity](images/long-stay-baseline/sparsity.jpeg)
```

Comment:
y-axis: the row information doesn't seems like too important. May be can change it to something like `case_id`/`case_index`? Also can reduce the number of y-ticks location (e.g., just show, 1, 100000, 200000, etc.). Now it looks a bit too many numbers listed on the y-axis.